### PR TITLE
Tweak: Use `aspect-ratio` property instead of history CSS trick [ED-8875]

### PR DESCRIPTION
**Pull request by arielk**

### CODE REVIEW

File: assets/dev/scss/global/global.scss [hunk 1]
```diff
@@ -1,7 +1,6 @@
 @import "../../../../assets/dev/scss/helpers/variables";
 @import "../../../../assets/dev/scss/editor/breakpoints";
 @import "dialog";
-@import "aspect_ratio";
```
Comment (StyleShit) [inline assets/dev/scss/global/global.scss:4]: Will break new Pro versions that still uses the aspect-ratio classes.
Maybe keep those classes and just change the implementation, or at least keep this file for BC (2-3 major versions?)
And what about 3rd party? do they use those classes?

Comment (arielk) [reply assets/dev/scss/global/global.scss:4]: @StyleShit In this case I'm pretty good with the backward compatibility. But I partially revert just to be safe until the next version. Please see the changes : https://github.com/elementor/elementor/pull/20718/commits/4e71425afb82b9bea3da2659fe0540e62481dfa3

File: assets/dev/scss/editor/panel/controls/_media.scss [hunk 2]
```diff
@@ -130,13 +133,14 @@
...
-		background-color: rgba(#fff, .9);
+		color: #fff;
+		background-color: rgba($editor-dark, .85);
 		border-radius: 3px;
 		box-shadow: 0 0 7px 0 rgba(0, 0, 0, 0.1);
+		transition: background .3s;
 
 		&:hover {
-			background-color: rgba(#fff, 1);
+			background-color: rgba($editor-dark, .95);
```
Comment (StyleShit) [inline assets/dev/scss/editor/panel/controls/_media.scss:143]: how are all those color changes related to the aspect ratio change?
should be in a different PR

File: assets/dev/scss/editor/panel/controls/_media.scss [hunk 3]
```diff
@@ -130,13 +133,14 @@
 		width: 20px;
 		height: 20px;
 		font-size: 11px;
-		color: $editor-light;
-		background-color: rgba(#fff, .9);
+		color: #fff;
```
Comment (StyleShit) [inline assets/dev/scss/editor/panel/controls/_media.scss:136]:
```suggestion
		color: $white;
```

File: assets/dev/scss/frontend/widgets/video.scss [hunk 4]
```diff
@@ -8,6 +8,17 @@
...
+
+		iframe,
+		video {
+			height: 100%;
+			width: 100%;
+			border: none; // To deal with inconsistencies
+			background-color: #000;
+		}
+	}
+
```
Comment (StyleShit) [inline assets/dev/scss/frontend/widgets/video.scss:21]: same as above, is it related to this PR?

File: assets/dev/scss/frontend/widgets/video.scss [hunk 5]
```diff
@@ -8,6 +8,17 @@
...
 	}
 
+	.elementor-wrapper {
+
+		iframe,
+		video {
+			height: 100%;
+			width: 100%;
+			border: none; // To deal with inconsistencies
+			background-color: #000;
```
Comment (StyleShit) [inline assets/dev/scss/frontend/widgets/video.scss:18]: why do you need this black background?

File: assets/dev/scss/global/_aspect_ratio.scss [hunk 6]
```diff
@@ -1,3 +1,5 @@
+
+// TODO: Remove this method because it has been replaced by native aspect ratio property in v3.11
```
Comment (StyleShit) [inline assets/dev/scss/global/_aspect_ratio.scss:2]:
```suggestion
// TODO: Remove this method because it has been replaced by native aspect ratio property in v3.11
```

Comment (StyleShit) [reply assets/dev/scss/global/_aspect_ratio.scss:2]: Maybe put it in the backlog, so we'll actually do it.
Plus, maybe we need to tell 3rd party that we're gonna remove it?

File: assets/dev/scss/global/global.scss (hunk 1 above)
Comment (StyleShit) [inline assets/dev/scss/global/global.scss:4]: why did you move it? can't we keep it here?

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: assets/dev/scss/global/_aspect_ratio.scss (hunk 6 above)
Comment (arielk) [reply assets/dev/scss/global/_aspect_ratio.scss:2]: Sure.

File: assets/dev/scss/global/global.scss (hunk 1 above)
Comment (arielk) [reply assets/dev/scss/global/global.scss:4]: In the past its loading twice in frontend.css & common.css because we want to use it also in the editor, now it's just for a BC in the front

File: assets/dev/scss/frontend/widgets/video.scss (hunk 5 above)
Comment (arielk) [reply assets/dev/scss/frontend/widgets/video.scss]: Added a comment

**Review by StyleShit — DISMISSED** [review]

File: assets/dev/scss/frontend/frontend-lite.scss [hunk 7]
```diff
@@ -1,4 +1,5 @@
 @import "../global/global";
+@import "../global/aspect_ratio"; //TODO: Deprecated in v3.11
```
Comment (KingYes) [inline assets/dev/scss/frontend/frontend-lite.scss:2]:
```suggestion
@import "../global/aspect_ratio"; // TODO: Deprecated in v3.11
```
